NetSDK/Services/EngineEvaluator/Splitter.cs

```diff
@@ -57,7 +57,7 @@ public long Hash(string key, int seed)
             byte[] seedResult = hashAlgorithm.ComputeHash(keyToBytes, 0, keyToBytes.Length);
             var result = BitConverter.ToInt32(seedResult, 0);
 
-            return result;
+            return result & 0xFFFFFFFFL;
```

Comment:
var result = BitConverter.ToInt32(seedResult, 0); => here result is Int32 (signed int, from -2,147,483,648 to 2,147,483,647)
At the end, "result & 0xFFFFFFFFL" is Int64 (or long) , signed, from -9,223,372,036,854,775,808 to 9,223,372,036,854,775,807